backend/src/test/java/com/ez/ezbackend/budget/request/ScheduleRequestTest.java

```diff
@@ -121,25 +130,29 @@ private void testConvertToScheduleWith(User user, Category category, Long updati
         .recurringPattern(RecurringPattern.YEARLY)
         .startDate(LocalDate.now())
         .build();
-    Schedule schedule = null;
-    if (category == null && updatingScheduleId == null) {
-      schedule = ScheduleRequest.convertToSchedule(scheduleRequest, user);
-    } else if (category != null && updatingScheduleId == null) {
-      schedule = ScheduleRequest.convertToSchedule(scheduleRequest, user, category);
-    } else if (category != null && updatingScheduleId != null) {
-      schedule = ScheduleRequest.convertToSchedule(scheduleRequest, user, category, updatingScheduleId);
-    }
+
+    Schedule previousSchedule = Schedule.builder()
+        .id(updatingScheduleId)
+        .startDate(LocalDate.of(2018, 1, 1))
+        .nextRecurringDate(LocalDate.of(2018,1,15))
```

Comment:
nitpicking, spaces after comma in `2018,1,15`

Reply:
Fixed.